crates/driver/example.toml

```diff
@@ -5,6 +5,9 @@ absolute-slippage = "40000000000000000" # Denominated in wei, optional
 relative-slippage = "0.1" # Percentage in the [0, 1] range
 account = "0x0000000000000000000000000000000000000000000000000000000000000001" # The private key of the solver
 
+[solver.request-headers]
+authorization = "FAKE_AUTH_TOKEN"
```

Comment:
nit: Probably slightly nicer to make up some fake header name as well instead of using a header that actually has meaning. If some people use the example file they might run into hard to debug issues because of it.

Reply:
Fake header instead of `solver.request-headers` or instead of `authorization`? What would you prefer it be called?

Reply:
Instead of `authorization`. Maybe `custom-header-one = "custom-value-one"` to imply that you can add any value you want.

Reply:
ah that makes more sense.  No Problem